orbs/shared/jobs/test.yaml

```diff
@@ -58,6 +62,7 @@ environment:
   AWS_REGION: << parameters.aws_region >>
   PRE_SETUP_SCRIPT: << parameters.pre_setup_script >>
   GO_TEST_TIMEOUT: << parameters.go_test_timeout >>
+  GOMEMLIMIT: << parameters.gomemlimit >>
```

Comment:
issue: when you set this, the entire check for heuristically determining the value of `GOMEMLIMIT` is always bypassed due to this check:

https://github.com/getoutreach/devbase/blob/239fd4372b6e3d360cb351c000e96958b66025ef/shell/golangci-lint.sh#L44

We need to figure out a better way of allowing repos to specify the value without removing the heuristic calculations feature.